winui/Shadow/Customization.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: Customization in WinUI Shadow control | Syncfusion
+description: Learn here all about the customization features such as color, opacity, position, and blur radius in Syncfusion WinUI Shadow control.
+platform: WinUI
+control: SfShadow
+documentation: ug
+---
+
+# Customization in WinUI Shadow
+
+This section explains the customization features available in the WinUI Shadow control.
+
+## Applying Color for shadow
+
+The shadow color can be customized using the `Color` property. The default value of `Color` property is `Black` color with 25% alpha value.
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfShadow Color="Red">
+    <Button Height="50" width="100" Content="Button"/>
+</syncfusion:SfShadow>
+
+{% endhighlight %}
+{% highlight C# %}
+
+   SfShadow shadow = new SfShadow();
+   shadow.Color = Color.FromArgb(255, 255, 0, 0);
+   
+   Button button = new Button(); 
+   button.Height = 50;
+   button.width = 100;
+   button.Content = "Button";
+   shadow.Content = button;
+
+{% endhiglight %}
+{% endtabs %}
+
+![Color customization in WinUI Shadow control](Shadow_images/winui_shadow_color.png)
+
+## Applying BlurRadius for shadow
+
+Blur level of shadow effect can be customized using the `BlurRadius` property. The default value of `BlurRadius` property is 8.
```

Comment:
The blur level of the shadow effect can be customized using the `BlurRadius` property. The default value of `BlurRadius` property is 8.

Reply:
Changed the Blur Property content as mentioned above.